common/src/main/java/net/dodogang/plume/block/BeamBlock.java

```diff
@@ -22,33 +22,41 @@
 import java.util.List;
 import java.util.Map;
 
-@SuppressWarnings({"unused","deprecation"})
+@SuppressWarnings({"unused", "deprecation"})
 public class BeamBlock extends Block implements Waterloggable {
-    public static final BooleanProperty POST = BooleanProperty.of("post");
-    public static final BooleanProperty NORTH = Properties.NORTH;
-    public static final BooleanProperty EAST = Properties.EAST;
-    public static final BooleanProperty SOUTH = Properties.SOUTH;
-    public static final BooleanProperty WEST = Properties.WEST;
-    public static final BooleanProperty UP = Properties.UP;
-    public static final BooleanProperty DOWN = Properties.DOWN;
+    public static final BooleanProperty POST        = BooleanProperty.of("post");
+    public static final BooleanProperty NORTH       = Properties.NORTH;
+    public static final BooleanProperty EAST        = Properties.EAST;
+    public static final BooleanProperty SOUTH       = Properties.SOUTH;
+    public static final BooleanProperty WEST        = Properties.WEST;
+    public static final BooleanProperty UP          = Properties.UP;
+    public static final BooleanProperty DOWN        = Properties.DOWN;
     public static final BooleanProperty WATERLOGGED = Properties.WATERLOGGED;
 
-    public static final Map<Direction, BooleanProperty> PROP_MAP = Util.make(new HashMap<>(),
-        map -> {
-            map.put(Direction.NORTH, NORTH);
-            map.put(Direction.EAST, EAST);
-            map.put(Direction.SOUTH, SOUTH);
-            map.put(Direction.WEST, WEST);
-            map.put(Direction.UP, UP);
-            map.put(Direction.DOWN, DOWN);
-        });
+    public static final Map<Direction, BooleanProperty> PROP_MAP = Util.make(new HashMap<>(), map -> {
+        map.put(Direction.NORTH, NORTH);
+        map.put(Direction.EAST, EAST);
+        map.put(Direction.SOUTH, SOUTH);
+        map.put(Direction.WEST, WEST);
+        map.put(Direction.UP, UP);
+        map.put(Direction.DOWN, DOWN);
+    });
 
     private final ShapeUtil shapeUtil;
 
     public BeamBlock(AbstractBlock.Settings settings) {
         super(settings);
 
-        this.setDefaultState(this.getStateManager().getDefaultState().with(POST, true).with(NORTH, false).with(EAST, false).with(SOUTH, false).with(WEST, false).with(UP, false).with(DOWN, false).with(WATERLOGGED, false));
+        this.setDefaultState(this.getStateManager()
+                                 .getDefaultState()
+                                 .with(POST, true)
+                                 .with(NORTH, false)
+                                 .with(EAST, false)
+                                 .with(SOUTH, false)
+                                 .with(WEST, false)
+                                 .with(UP, false)
+                                 .with(DOWN, false)
+                                 .with(WATERLOGGED, false));
```

Comment:
I like this. But also, I manually did this. The formatter didn't remove it tho, so I think we might be able to format some of its bad decisions manually